src/main/java/gregtech/loaders/recipe/handlers/PipeRecipeHandler.java

```diff
@@ -51,7 +52,7 @@ private static void processRestrictivePipe(OrePrefix pipePrefix, Material materi
         else if (pipePrefix == OrePrefix.pipeHugeRestrictive) unrestrictive = OrePrefix.pipeHugeItem;
         else return;
 
-        RecipeMaps.ASSEMBLER_RECIPES.recipeBuilder()
+        RecipeMaps.PACKER_RECIPES.recipeBuilder()
```

Comment:
Why the change to use the Packager? It does not really make sense to use it here.